### DISCUSSION TITLE
interval doubling

### DISCUSSION THREAD

**Discussion by ayerix** [ReactiveX/rxjs#7328] [Q&A] [answered]
Please explain why this example doubling the output and how to avoid the problem

import { interval, Subject } from 'rxjs';

let x = new Subject();
x.asObservable().subscribe(t => console.log(t));
interval(5000).subscribe(x);

output is 00112233 etc

**Comment by voliva** [comment, marked as answer]
It shouldn't, and it doesn't https://codesandbox.io/s/rxjs-playground-3rkq8z?file=/src/index.ts

Are you sure your code snippet is running just once?

Otherwise it might be a bug of the rxjs version you're using.
